packages/ui-popover/src/Popover/index.tsx

```diff
@@ -192,6 +192,15 @@ class Popover extends Component<PopoverProps, PopoverState> {
     }
   }
 
+  isContentOutOfView() {
+    if (this._contentElement) {
+      const rect = this._contentElement.getBoundingClientRect()
+      const windowHeight = window.innerHeight
```

Comment:
I think trying to access `window` with SSR will lead to a crash? Can you please use `canUseDOM` and if its false this function could just return `false`.